lxd/device/nic_bridged.go

```diff
@@ -419,7 +419,7 @@ func (d *nicBridged) removeFilters(m deviceConfig.Device) error {
 
 			// If we get this far, then the current host rule matches one of our LXD
 			// rules, so we should run the modified command to delete it.
-			_, err = shared.RunCommand(fields[0], fields[1:]...)
+			_, err = shared.RunCommand(fields[0], append([]string{"--concurrent"}, fields[1:]...)...)
```

Comment:
@stgraber do you need to do the append here, how about just:

```
_, err = shared.RunCommand(fields[0], "--concurrent", fields[1:]...)
```

Reply:
Won't compile, RunCommand is picky about the args.

Reply:
@stgraber ah ok, I guess you cant mix normal and slice expanded args for a single variadic receiver.